docs/articles/infrastructure/ci-cd/build-slaves.rst

```diff
@@ -136,7 +136,7 @@ Replace <URL> in the file with the real URL to the Jenkins server.
 6. Save
 7. Click on that slave in the list to get to the online/offline view of it
 8. Copy the command line shown there and put it into the ``jenkins.slave.service`` file
-9. Download the linked ``slave.jar`` and put it into the home directory of the jenkins user so that the service can find it
+9. Download the linked ``agent.jar`` (``slave.jar`` for older versions) and put it into the home directory of the jenkins user so that the service can find it
```

Comment:
What is the reason for the file name change?

Reply:
Jenkins project changed it: https://issues.jenkins-ci.org/browse/INFRA-1105